src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -2,15 +2,32 @@ import { FC } from "react";
 import { MainTable } from "@canonical/react-components";
 import { LxdInstance } from "types/instance";
 import { getInstanceDevices } from "util/helpers";
+import { isRootDisk } from "util/instanceValidation";
+import { FormDevice } from "util/formDevices";
+import ResourceLink from "components/ResourceLink";
+import { isOtherDevice } from "util/devices";
 
 interface Props {
   instance: LxdInstance;
 }
 
 const InstanceOverviewDevices: FC<Props> = ({ instance }) => {
-  const instanceDevices = getInstanceDevices(instance).filter(([, device]) => {
-    return device.type !== "nic" && device.type !== "disk";
-  });
+  const instanceDevices = getInstanceDevices(instance)
+    .filter(([, device]) => {
+      return device.type !== "nic" && device.type !== "none";
+    })
+    // Sort devices by type and name
+    .sort((a, b) => {
+      const nameA = a[0].toLowerCase();
+      const nameB = b[0].toLowerCase();
+      const typeA = a[1].type;
+      const typeB = b[1].type;
+      if (nameA === nameB) {
+        return typeA.localeCompare(typeB);
+      }
+
+      return nameA.localeCompare(nameB);
```

Comment:
I meant the other way around -- first type and then name, so the disk devices are on top. Because I think the disks are the most relevant ones and grouping by type seems better than sorting by name.

Reply:
right, I read that wrong apologies